test/topology_experimental_raft/test_tablets.py

```diff
@@ -1222,3 +1240,206 @@ async def test_tablet_storage_freeing(manager: ManagerClient):
     logger.info("Verify that the table's disk usage on first node shrunk by about half.")
     size_after = await manager.server_get_sstables_disk_usage(servers[0].server_id, "test", "test")
     assert size_before * 0.33 < size_after < size_before * 0.66
+
+@pytest.mark.asyncio
+async def test_decommission_rack_basic(manager: ManagerClient):
+    """
+    Test decommissioning of all nodes in a rack
+    when there are enough remaining racks to satisfy
+    the replication factor.
+    """
+    logger.info("Bootstrapping multi-rack cluster")
+    servers = dict[ServerNum, ServerInfo]()
+    num_racks = 3
+    nodes_per_rack = 2
+    shards_count = 2
+    initial_tablets = 64
+    rf = num_racks - 1
+    ks = "test"
+    table = "test"
+    for rack in range(1, num_racks + 1):
```

Comment:
This looks like a good utility function `servers = await create_racks(racks_count, nodes_per_rack, property_file)`

Reply:
I can add it in this source file.  Not sure it is fit for a general purpose utility function.
Also the property file would be generated by the function, not passed to it.
We can pass the dc name (with e.g. `dc1` as default value)

Reply:
> Not sure it is fit for a general purpose utility function.

I suspect we will have more testcases where we need to setup entire rack in the future so I can see how that function would be useful